spec/models/case/sar/offender_spec.rb

```diff
@@ -76,6 +76,15 @@
     end
   end
 
+  describe "#external_deadline for rejected" do
+    let(:rejected_offender_sar_case) { create(:offender_sar_case, :rejected) }
+    let(:received_date) { Time.zone.today }
+
+    it "sets external deadline using a calculation of 90 days which is extended to next working day" do
```

Comment:
In what way is it extended to the next working day?

Reply:
It doesn't extend the date to the next working day therefore removed from description.